rxjs-1/speed.js

```diff
@@ -0,0 +1,31 @@
+const { fromEvent } = require('rxjs');
+const { throttleTime, flatMap } = require('rxjs/operators');
+
+const { getRace } = require('./race');
+const { round } = require('./util');
+
+const race = getRace();
+const carNames = race.getCars();
+
+console.log('Participants:', carNames.join('/'));
+
+const ONE_SEC_IN_MS = 1e3
+
+function getCarSpeed(race, carName) {
+  return fromEvent(race, 'data').pipe(
+    throttleTime(200),
+    flatMap(car => car.carName === carName
```

Comment:
Could a simple `map` do the trick ? Why ?

Reply:
A `map` would just transform the array model. The goal here is about to `filter` on the array then `map` on the matching result. 
So without `flatMap` I would use those two operators.